README.md

```diff
@@ -8,6 +8,7 @@ The upload is enriched with metadata from the github api and the README.md
 
 Prerequisites (with debian or ubuntu):
 
+    apt update && apt upgrade
```

Comment:
the update of all the installed packages is not a requirement for installing iagitup.